src/java.base/share/man/java.1

```diff
@@ -5533,6 +5533,46 @@ application start\-up script with the same technique:
 .PP
 \f[CB]env\ JAVA_TOOL_OPTIONS=\-XX:+RecordDynamicDumpInfo\ bash\ app_start.sh\f[R]
 .RE
+.SS Creating Dynamic CDS Archive File with \-XX:+AutoCreateSharedArchive
+.PP
+\f[CB]\-XX:+AutoCreateSharedArchive\f[R] is a more convenient way of
+creating/using CDS archives.
+Unlike the methods of manual CDS archive creation described in the
+previous section, with \f[CB]\-XX:+AutoCreateSharedArchive\f[R], it\[aq]s
+no longer necessary to have a separate trial run.
+Instead, you can always run the application with the same command\-line
+and enjoy the benefits of CDS automatically.
+.RS
+.PP
+\f[CB]java\ \-XX:+AutoCreateSharedArchive\ \-XX:SharedArchiveFile=hello.jsa\ \-cp\ hello.jar\ Hello\f[R]
+.RE
+.PP
+If the specified archive file does not exist, it will be automatically
+created at VM exit.
+.PP
+If the specified archive file exists, then the behavior depends on the
+version of the JDK that created this archive:
+.IP \[bu] 2
+If the archive was created by a JDK prior to version 19, it will be
+ignored and will not be loaded.
+Nor will it be updated.
+.IP \[bu] 2
+If the archive was created by the same version of the JDK as the current
+JVM, it will be loaded as a dynamic archive.
+The archive will not be updated at VM exit.
+.IP \[bu] 2
+In all other cases, the JVM will not load the archive, but will update
+the archive at exit.
+Thus, the archive will be ready to be loaded the next time the JVM is
+launched with the same command line.
```

Comment:
This still seems rather complicated. Suggestion:

> 
> If the specified archive file exists and was created by the same version of the JDK, then it will be loaded as a dynamic archive; otherwise it is ignored at VM startup.
> 
> At VM exit, if the specified archive file does not exist, it will be created. If it exists but was created with a different (but post JDK 19) version of the JDK, then it will be replaced. In both cases the archive will be ready to be loaded the next time the JVM is launched with the same command line.
> 
> If the specified archive file exists but was created by a JDK version prior to JDK 19, then it will be ignored: neither loaded at startup, nor replaced at exit.
>